blockchain/types/block.go

```diff
@@ -63,21 +63,32 @@ type Header struct {
 	Governance []byte `json:"governanceData"            gencodec:"required"`
 	Vote       []byte `json:"voteData,omitempty"`
 
-	BaseFee *big.Int `json:"baseFeePerGas,omitempty"    rlp:"optional"`
+	// Added with Magma hardfork for KIP-71 dynamic fee.
+	BaseFee *big.Int `json:"baseFeePerGas,omitempty" rlp:"optional"`
+
+	// Added with Randao hardfork for KIP-114 RANDAO.
+	RandomReveal []byte `json:"randomReveal,omitempty" rlp:"optional"` // 96 byte BLS signature
```

Comment:
Why not use array `[96]byte` rather slice. Or, a new type may be introduced like `common.hash`